Specs/ASTReduceSpec.m

```diff
@@ -43,6 +47,17 @@ - (void)testEverything {
 
                     XCTAssertEqualObjects(ASTReduce(dict, @4, add), @10);
                 }];
+                
+                [XCTContext runActivityNamed:@"should be able to reduce key value pairs in dictionary" block:^(id<XCTActivity> _Nonnull activity){
```

Comment:
Can you move this out of the `for objects implementing <NSFastEnumeration>` scope?